documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -167,6 +167,9 @@ public List<FileNode> getFilesTimeline(DocumentTimelineFilter filter,
         String workspace = session.getWorkspace().getName();
         String sortField = getSortField(filter, false);
         String sortDirection = getSortDirection(filter);
+        if (rootPath.endsWith(username + "/Private")) {
```

Comment:
I am not sure we need this change, the Private folder has already a symlink to the /Public folder. it should be another cause for filtering files from /Public files, could you check again ?